src/views/Dashboard/Dashboard.jsx

```diff
@@ -1,44 +1,58 @@
-import React, {useState} from "react";
+import React, {Component} from "react";
 import Sidebar from "../../components/Sidebar";
 import {DEFAULT_ACTIVE_STATE} from "../../components/Sidebar/Sidebar.jsx";
 import BillDisplay from "../BillDisplay";
+import Profile from "../Profile";
 import "./styles.scss";
 
-export default ({history,userInfo}) => {
-  //testing with @testing-library to be learnt later
-  const [activeState, setActiveState] = useState(
-                                                {
-                                                  ...DEFAULT_ACTIVE_STATE,
-                                                  bills: true
-                                                }
-                                              );
-  const [component, setComponent] = useState(<BillDisplay />);
+export default class Dashboard extends Component {
```

Comment:
ok im going to pause reviewing. My refactoring changes will definately have conflicts with this change

Reply:
This is not the latest commit https://github.com/dannylekim/billsnapFront/blob/feat/%23a4k5en_%23a6k6a5/src/views/Dashboard/Dashboard.jsx